broker/config/settings.yml

```diff
@@ -187,6 +187,12 @@ defaults: &defaults
       key: <%= certificate('broker.key') %>
       cert: <%= certificate('broker.crt') %>
 
+  apiserver:
+    ip: '127.0.0.1'
+    port: 9443
+    encryption:
+      password: 'ABCDEFGHABCDEFGHABCDEFGHABCDEFGH'
```

Comment:
you can just name the parameter 'password' --> 'key' as to be consistent with **crypto.createCipheriv** and **crypto.createDecipheriv** methods' parameters-naming.